CHANGELOG.md

```diff
@@ -60,6 +60,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `soil_drylimit_factor` to the DU2G_instance_DU.rc and DU2G_GridCompMod.F90 files for FENGSHA
 - Moved process library macros to header file.
 
+## [#263] - 2024-02-02
+
+### Changed
+-Sulfate surface area density calculation in SU_Compute_Diags was incorrectly being passed the effective radius used for settling along with the sigma width of the number distribution. Properly it should be passed the number median radius, also present in the RC file. Added a hook to read that field from the RC file ("particle_radius_number"), store in SU grid comp, and pass to SU_Compute_Diags. This change is zero-diff to the SU internal state. It changes value of export SO4AREA.
+
+-Add some protective logic around reading daily volcanic emission files. If filename does not exist model will write message to standard output and reset volcanic emissions to zero. This is zero difference result unless in a place where the files were not present.
```

Comment:
Does this part still holds relevance because the change in the code does not seem to indicate a test for the file's existence?